docs/apis-tools/testing/getting-started.md

```diff
@@ -105,14 +107,15 @@ import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
 @CamundaSpringProcessTest
+@Deployment(resources = "classpath:my-process.bpmn")
```

Comment:
:wrench: Usually, the `@Deployment` annotation is at the process application class. 

The Spring test case starts the process application that deploys the processes.